src/Validator/Authentication.php

```diff
@@ -68,6 +68,17 @@ class Authentication extends AbstractValidator
      */
     protected $service;
 
+    /**
+     * Authentication\Result codes mapping
+     * @var array
+     */
+    protected static $codeMap = [
+        Result::FAILURE_IDENTITY_NOT_FOUND => self::IDENTITY_NOT_FOUND,
+        Result::FAILURE_CREDENTIAL_INVALID => self::CREDENTIAL_INVALID,
+        Result::FAILURE_IDENTITY_AMBIGUOUS => self::IDENTITY_AMBIGUOUS,
+        Result::FAILURE_UNCATEGORIZED      => self::UNCATEGORIZED,
+    ];
```

Comment:
I'd argue this should be a private constant (which we can do as we're now targeting PHP 5.6 and up in develop). I'll do this and push to your branch.